service/docs/source/active_session.schema.json

```
@@ -0,0 +1,35 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "https://geopm.github.io/active_session.schema.json",
+    "title": "ActiveSession",
+    "type": "object",
+    "properties": {
+      "client_pid": {
+        "type": "number"

```

```suggestion
        "type": "integer"
```
The PID will always be an integer.  I don't think we need to support anything else here.
https://json-schema.org/understanding-json-schema/reference/numeric.html

---

I see that this was done just to match what's already in system_files.py for the ActiveSessions schema.  I'm wondering if we should change this there as well.

---

> I see that this was done just to match what's already in system_files.py for the ActiveSessions schema. I'm wondering if we should change this there as well.

Yeah, I noticed that and opted to just follow what was there for now. But this may be a good opportunity to fix it in the code, then.